autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/ExpandedCourseCard.tsx

```diff
@@ -115,16 +115,17 @@ const ExpandedCourseCard: React.FC<ExpandedCourseCardProps> = ({
           value={course}
           multiple={false}
           filterOptions={(): any[] => options} // Options are not filtered
+          getOptionSelected={(option): boolean => option === options[0]}
```

Comment:
```suggestion
          getOptionSelected={() => true}
```
Shouldn't this do the same thing

Reply:
Yeah, you're right. I thought it called this function on all of the options, but it turns out it only applies it to the current value of the Autocomplete. Fixed as of 7/25